resources/validations/src/ssp.sch

```diff
@@ -18,6 +18,9 @@
     <xsl:param as="xs:string"
                name="registry-base-path"
                select="'../../xml'" />
+    <xsl:param as="xs:string"
+               name="baselines-base-path"
+               select="'../../../src/baselines/rev4/xml'" />
```

Comment:
B: I would recommend not using `src` paths and the finalized content for reasons I will explain further below.

```suggestion
               select="'../../../baselines/rev4/xml'" />
```